specification/databox/resource-manager/Microsoft.DataBox/stable/2020-11-01/databox.json

```diff
@@ -2299,7 +2299,7 @@
       "required": [
         "storageLocation",
         "skuName",
-        "expectedDataSizeInTerabytes"
+        "expectedDataSizeInTeraBytes"
```

Comment:
It was complaining because terabytes isn't camelCase

Reply:
Hi Nick, we need to include this word somehow to avoid S360 and due to some other dependencies we can't move back to previous word. If adding to custom words is not the right way, Kindly guide me on possible solution ?

Reply:
things that are actually typos, but can't be fixed in a particular version can be ignore on a per-file basis like https://github.com/Azure/azure-rest-api-specs/blob/8c8ae024cbd209644e6f93a3bb031f9cffb07401/cSpell.json#L41-L44

Reply:
> things that are actually typos, but can't be fixed in a particular version can be ignore on a per-file basis like
> 
> https://github.com/Azure/azure-rest-api-specs/blob/8c8ae024cbd209644e6f93a3bb031f9cffb07401/cSpell.json#L41-L44

Thank you, Just as in case user use different version, we will get swagger correctness issue on that version, Can I change spell of this property in all previous version and also add tera to ignore for all those, as part if this PR @nschonni

Reply:
Looks like there is one done that way https://github.com/Azure/azure-rest-api-specs/blob/8c8ae024cbd209644e6f93a3bb031f9cffb07401/cSpell.json#L686-L689
Not sure what the MS process is around that stuff

FYI, reported this a few years ago https://github.com/Azure/azure-rest-api-specs/issues/4670, but it got closed with no resolution

Reply:
Added this change for latest version. @nschonni, Kindly check, review and if good please approve. Thanks.